app/src/main/java/com/oracle/javafx/scenebuilder/app/DocumentWindowController.java

```diff
@@ -128,7 +128,7 @@
 /**
  * Handles all editing and control actions in the Scene Builder editor window.
  * Editing action are used to modify a document (cut, paste, delete, import).
- * Control actions in most cases do not cause document structure modifications. 
+ * Control actions in most cases do not cause document structure modifications.
  */
```

Comment:
There seem to be quite a large number of changes in this file, possibly due to autoformatting of some sort. Could we revert changes to this file please and see if there are non-formatting changes here.